server/polar/subscription/schemas.py

```diff
@@ -72,3 +78,50 @@ class SubscriptionCreateEmail(Schema):
     product_id: UUID4 = Field(
         description="The ID of the product. **Must be the free subscription tier**."
     )
+
+
+class SubscriptionCancel(Schema):
+    immediately: bool = Field(
```

Comment:
I don't really like that naming. Maybe we could use Stripe naming `cancel_at_period_end`?

Reply:
Agree. I think I'll change it to `now=True` for immediate cancellation perhaps. 

**Immediate**
`polar.subscriptions.cancel(id, { now: true })`

**Period end (default)**
`polar.subscriptions.cancel(id)`

Feels like that reads well. `cancel_at_period_end` is intuitive as part of the `subscription` object, but feels a bit verbose and counterintuitive with `cancel_at_period_end=False` to flag immediate as part of the cancel method.